script.js

```diff
@@ -35,47 +35,128 @@ function getHumanChoice() {
+      gameData.classList.add("gameData");
+      gameData.textContent = "Tie! None gets a point";
+      resultDiv.appendChild(gameData);
+      // resultDiv.textContent = "Tie! None gets a point";
+    } else {
+      console.log("Error, combination not yet added");
+      gameData.classList.add("gameData");
+      gameData.textContent = "Error, combination not yet added";
+      resultDiv.appendChild(gameData);
+      // resultDiv.textContent = "Error, combination not yet added"
+    }
   }
+function playGame(humanButtonSelection) {
+  // Declare score values
+  // let humanScore = 0;
+  // let computerScore = 0;
+  // Loop function, run it 5 times
+  // for (let step = 0; step <5; step++) 
+  {
+  // Function that has the main game logic
   // Play game
-  const humanSelection = getHumanChoice();
+  playRound();
+  const humanSelection = humanButtonSelection;
   const computerSelection = getComputerChoice();
   // Print choices
   console.log(`Computer chose ${computerSelection} \n\Human chose ${humanSelection}`);
+  gameData.classList.add("gameData");
+  gameData.textContent = `Computer chose ${computerSelection} \n\Human chose ${humanSelection}`;
+  resultDiv.appendChild(gameData);
+  // resultDiv.textContent = `Computer chose ${computerSelection} \n\Human chose ${humanSelection}`;
   playRound(humanSelection, computerSelection);
   }
-  if (humanScore > computerScore)
-    console.log("Human won the game, congratulations!");
-  else if (computerScore > humanScore)
-    console.log("Computer won the game, better luck next time human!");
-  else console.log("It's a tie. Give it another go to get a winner!");
+  if (humanScore >= 5 || computerScore >= 5) {
+    if (humanScore > computerScore) {
+      console.log("Human won the game, congratulations!");
+      gameData.classList.add("gameData");
+      gameData.textContent = "Human won the game, congratulations!";
+      resultDiv.appendChild(gameData);
+      // resultDiv.textContent = "Human won the game, congratulations!";
+    }
+    else if (computerScore > humanScore) {
+      console.log("Computer won the game, better luck next time human!");
+      gameData.classList.add("gameData");
+      gameData.textContent =
+        "Computer won the game, better luck next time human!";
+      resultDiv.appendChild(gameData);
+      // resultDiv.textContent = "Computer won the game, better luck next time human!";
+    }
+    else {
+      console.log("Computer won the game, better luck next time human!");
+      gameData.classList.add("gameData");
+      gameData.textContent = "It's a tie. Give it another go to get a winner!";
+      resultDiv.appendChild(gameData);
+      // resultDiv.textContent = "It's a tie. Give it another go to get a winner!";
+    }
+  }
+}
+// playGame(humanButtonChoice);
+// console.log("Game end");
+const rockbtn = document.querySelector("#btn-rock");
+function onRockClick(){
+  // console.log("Rock btn");
+  playGame("rock");
+}
```

Comment:
Here, each of your `onXClick` only calls `playGame("X")`. So you could remove the need for `onXCall` altogether and call `playGame("X")` in the HTML.